packages/example/src/pages/Home.tsx

```diff
@@ -23,6 +26,8 @@ export const Home = () => {
             <ProgramListWithTitle title="Watch again" />
             <Spacer gap="$6" />
             <ProgramListWithTitle title="You may also like..." />
+            <Spacer gap="$6" />
+            <ProgramListWithTitleAndVariableSizes title="Our selection"></ProgramListWithTitleAndVariableSizes>
```

Comment:
Really a small detail

```suggestion
            <ProgramListWithTitleAndVariableSizes title="Our selection" />
```